### PR TITLE
Fixes : #480 Added Dark mode logo in Hero

### PR DESCRIPTION
**Pull request by siddharth9300**
Fixes : #480 Added Dark mode logo in hero section by creating different logo for dark mode and light mode.

Dark Mode : 
image

Light mode : 
image

### CODE REVIEW

**Comment by origranot** [comment]
@siddharth9300 Can't we do all of it with the fill property, what do you think?

**Comment by siddharth9300** [comment]
> @siddharth9300 Can't we do all of it with the fill property, what do you think?

Yes we can i please check the latest changes.

**Comment by origranot** [comment]
> > @siddharth9300 Can't we do all of it with the fill property, what do you think?
> 
> 
> 
> @origranot  Yes we can ,please check the latest changes. I have made changes and done it with the fill property.

I checked it. Can't we use the same svg logo and just pass the fill property? (instead of duplicate the svg code)

**Comment by siddharth9300** [comment]
@origranot Instead of using fill property I have used filter property. I have committed the changes.

**Comment by origranot** [comment]
@siddharth9300 Hey, can you please run the formatter to fix the prettier issues?

**Comment by siddharth9300** [comment]
@origranot Done 👍

File: apps/frontend/src/routes/index.tsx [hunk 1]
```diff
@@ -59,7 +62,7 @@ export default component$(() => {
             <div class="flex flex-col">
               <article class="prose mx-auto max-w-4xl pb-16">
                 <div class="mx-auto">
-                  <img class="mx-auto" src="logo.svg" width="410" height="73" alt="Logo" />
+                  <img class="mx-auto my-8" style={logoStyle} width="410" height="73" src="logo.svg" alt="Logo" />{' '}
```
Comment (origranot) [inline apps/frontend/src/routes/index.tsx:65]: ```{' '}``` Why do you need this?

**Review by origranot — CHANGES_REQUESTED** [review]
@siddharth9300 Great job!
I added few comments, what do you think?

File: apps/frontend/src/routes/index.tsx [hunk 2]
```diff
@@ -51,6 +51,9 @@ export default component$(() => {
 
   useContextProvider(InputContext, state);
 
+  const logoStyle = {
+    filter: globalStore.theme === 'light' ? 'invert(0)' : 'invert(1)',
+  };
```
[the remark this answers is not in the document]
Comment (siddharth9300) [reply apps/frontend/src/routes/index.tsx]: @origranot done👍

**Pull request merged**